student-store-ui/src/components/SubNavBar/SubNavBar.jsx

```diff
@@ -12,15 +12,15 @@ export default function SubNavBar({setSelectedCategory, searchInput, setSearchIn
                         <input type="text" 
                                placeholder="Search"
                                value={searchInput}
-                               onChange={ (event) => setSearchInput(event.target.value)}
+                               onChange={ (e) => setSearchInput(e.target.value)}
```

Comment:
nit: Preferred how it was before
`onChange={ (event) => setSearchInput(event.target.value)}`